docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.1'
+
+services:
+
+  # Build from local "Dockerfile" file
+  local-example:
+    build:
+      context: .
+    image: munkyboy/fakesmtp
+    volumes:
+      - /tmp/fakemail:/var/mail
+    ports:
+      - "2525:25"
```

Comment:
I prefer to not hardcode host ports to avoid conflicts with other services. Is it ok if you remove the `2525` mapping?

Reply:
Yes, I can remove it! I only copy your example as `docker-compose` format